traffic_ops/experimental/webfront/webfront.go

```diff
@@ -122,188 +145,311 @@ func main() {
 	Logger.Fatal(http.ListenAndServeTLS(":" + strconv.Itoa(int(config.ListenPort)), "server.pem", "server.key", s))
 }
 
-func validateToken(tokenString string) (*jwt.Token, error) {
-
-	tokenString = strings.Replace(tokenString, "Bearer ", "", 1)
-	token, err := jwt.ParseWithClaims(tokenString, &Claims{}, func(token *jwt.Token) (interface{}, error) {
-		if _, ok := token.Method.(*jwt.SigningMethodHMAC); !ok {
-			return nil, fmt.Errorf("Unexpected signing method: %v", token.Header["alg"])
-		}
-		return []byte(os.Args[2]), nil
-	})
-	return token, err
-}
-
-// NewServer constructs a Server that reads rules from file with a period
-// specified by poll.
+// NewServer constructs a Server that reads Rules from file with a period 
+// specified by poll
 func NewServer(file string, poll time.Duration) (*Server, error) {
```

Comment:
Polling to reload configs is unpredictable, and prone to error if a user is in the middle of editing a config. The standard method is via NOHUP signal, which most service management systems will send on `service reload`.

This is ok for /experimental, but you should plan on changing it to Nohup before it's moved out. See https://github.com/apache/incubator-trafficcontrol/blob/8eda1c7f510c1a6504ddf2ef125ba14aaccd523a/traffic_stats/traffic_stats.go#L137 for an example in Go.